qiskit/tools/monitor/overview.py

```diff
@@ -96,55 +94,54 @@ def backend_monitor(backend):
         return
 
     print()
-    qubit_header = 'Qubits [Name / Freq / T1 / T2 / U1 err / U2 err / U3 err / Readout err]'
-    print(qubit_header)
-    print('-'*len(qubit_header))
-
+    props = backend.properties()
+    qubit_header = None
     sep = ' / '
-    for qub in range(len(props['qubits'])):
-        name = 'Q%s' % qub
-        qubit_data = props['qubits'][qub]
-        gate_data = [g for g in props['gates'] if g['qubits'] == [qub]]
-        t1_info = qubit_data[0]
-        t2_info = qubit_data[1]
-        freq_info = qubit_data[2]
-        readout_info = qubit_data[3]
-
-        freq = str(round(freq_info['value'], 5))+' '+freq_info['unit']
-        T1 = str(round(t1_info['value'],
-                       5))+' ' + t1_info['unit']
-        T2 = str(round(t2_info['value'],
-                       5))+' ' + t2_info['unit']
-        for gd in gate_data:
-            if gd['gate'] == 'u1':
-                U1 = str(round(gd['parameters'][0]['value'], 5))
-                break
 
-        for gd in gate_data:
-            if gd['gate'] == 'u2':
-                U2 = str(round(gd['parameters'][0]['value'], 5))
-                break
-        for gd in gate_data:
-            if gd['gate'] == 'u3':
-                U3 = str(round(gd['parameters'][0]['value'], 5))
-                break
+    for index, qubit_data in enumerate(props.qubits):
+        name = 'Q%s' % index
+        gate_data = [gate for gate in props.gates if gate.qubits == [index]]
 
-        readout_error = str(round(readout_info['value'], 5))
+        cali_data = dict.fromkeys(['T1', 'T2', 'frequency', 'readout_error'], 'Unknown')
+        for nduv in qubit_data:
+            if nduv.name in cali_data.keys():
+                cali_data[nduv.name] = format(nduv.value, '.5f') + ' ' + nduv.unit
+
+        gate_names = []
+        gate_error = []
+        for gd in gate_data:
+            if gd.gate in ['id', 'rz']:
+                continue
+            for gd_param in gd.parameters:
+                if gd_param.name == 'gate_error':
+                    gate_names.append(gd.gate.upper() + ' err')
+                    gate_error.append(format(gd_param.value, '.5f'))
+
+        if not qubit_header:
+            qubit_header = 'Qubits [Name / Freq / T1 / T2 / ' + sep.join(gate_names) + \
+                           ' / Readout err]'
```

Comment:
Will this have a `T2 / / Readout err` substring if no gates are present?

Reply:
fixed in b53a983